app/helpers/displayHelpers.php

```diff
@@ -4797,3 +4797,27 @@ function caGetDisplayLabelForBundle($bundle) {
 		return $t->getDisplayLabel($bundle);
 	}
 	# ------------------------------------------------------------------
+	/**
+	 * Returns concatenated first name and surname, with a default in case
+	 * both values are null.
+	 *
+	 * In case any of the names is missing, no additional (left or right)
+	 * whitespace is included.
+	 *
+	 * @param      $fname
+	 * @param      $lname
+	 * @param null $default
+	 *
+	 * @return mixed|string
+	 */
+	function caDisplayName($fname, $lname, $default=null){
```

Comment:
Can you give this a more specific name – something more descriptive? We use "display" and "name" (and "displayname") in too many contexts, so this might be confusing as a helper.

Reply:
Sure, what about `caDisplayFullName`? Any other suggestion?

Reply:
How about caFormatPersonName() ?

Reply:
Sure!